timed/authentication.py

```diff
@@ -29,42 +26,39 @@ def get_introspection(self, access_token, id_token, payload):
             verify=settings.OIDC_VERIFY_SSL,
             headers=headers,
             data={"token": access_token},
+            timeout=10,
```

Comment:
if we ever reach this, then o boy are we sol

in the long run there might be potential as to providing lower order primaries lie retry-backoff implementation that would help alleviate why we need to set a timeout now